install/kubernetes/cilium/charts/hubble-relay/templates/validation.yaml

```diff
@@ -0,0 +1,3 @@
+{{- if not .Values.global.hubble.enabled }}
```

Comment:
should it be `{{- if and .Values.global.hubble.relay.enabled (not .Values.global.hubble.enabled) }}` ?

Reply:
oh sorry, I didn't notice that you changed to draft :(

Reply:
I converted to draft since it was failing helm lint. Your comment was helpful though, seems like this was indeed needed to pass the lint! Thanks! :)

In terms of deployment, the `hubble-relay` chart will not be included in a `cilium` deployment if 
`.Values.global.hubble.relay.enabled=false` (in the requirements.yaml)

Reply:
I just noticed that the current setup didn't check for sub-charts :), let me send PR shortly

Only parent chart is checked.
```
==> Linting install/kubernetes/cilium
[INFO] Chart.yaml: icon is recommended

1 chart(s) linted, 0 chart(s) failed
```